builtin/docker/platform.go

```diff
@@ -503,6 +525,16 @@ deploy {
 		),
 	)
 
+	doc.SetField(
+		"resources",
+		"A map of resources to configure the container with, such as memory or cpu limits.",
```

Comment:
nit
```suggestion
		"A map of resources to configure the container with, such as memory or cpu limits",
```

Reply:
Looking at this, it seems like some of our fields have periods, and others don't. But ultimately the docs generators adds them to the end when they get generated for the website, so maybe I'll leave this one here? Otherwise it's kind of confusing if it adds it back.